org.blender.Blender.json

```diff
@@ -153,7 +158,12 @@
                     "type": "archive",
                     "url": "https://ftp.nluug.nl/pub/graphics/blender/release/Blender2.91/blender-2.91.2-linux64.tar.xz",
                     "sha256": "8f1e1e8852750e1038579336c7461c1a5492da973ce188e1e5cae99b2f796a23",
-                    "strip-components": 0
+                    "strip-components": 0,
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 201,
+                        "url-template": "https://ftp.nluug.nl/pub/graphics/blender/release/Blender$version/blender-$version-linux64.tar.xz"
```

Comment:
Since the URL contains the version patterns `major.minor` and `major.minor.patch`, you currently have to use the HTML checker instead of the Anitya one: https://github.com/flathub/flatpak-external-data-checker/issues/134

Reply:
Wait, it works for sourceforge because they provide a rss feed. If there's no such feed/page, why not just build from source?

Reply:
The HTML and Anitya checkers now support more placeholders, like $major, $minor and $patch